.github/workflows/continuous_integration.yml

```diff
@@ -16,40 +16,52 @@ jobs:
       fail-fast: false
       matrix:
         config:
-
-          - name: "MSVC x64"
-            os: windows-latest
-            build_type: "Release"
-            vcpkg_triplet: x64-windows
-            shell: pwsh
-            extra_options: >-
-              -A x64
-              -DCMAKE_TOOLCHAIN_FILE="${env:VCPKG_INSTALLATION_ROOT}\scripts\buildsystems\vcpkg.cmake"
-              -DVCPKG_TARGET_TRIPLET=x64-windows
-
-          - name: "MSYS2 UCRT64"
-            os: windows-latest
-            build_type: "Release"
-            extra_options: "-G Ninja"
-            package_name: "mingw_x64"
-            shell: "msys2 {0}"
-            msystem: ucrt64
-            msys-env: mingw-w64-ucrt-x86_64
-            artifact-path: build/*.zip
-
-          - name: "Linux GCC"
-            os: ubuntu-latest
-            build_type: "Release"
-            extra_options: "-G Ninja -DCMAKE_INSTALL_PREFIX=/usr"
-            package_name: "linux_gcc"
-            shell: bash
-            artifact-path: build/*.appimage
-
-          - name: "macOS Clang"
-            os: macos-latest
-            build_type: "Release"
-            extra_options: "-G Ninja"
-            shell: bash
```

Comment:
https://github.com/kraflab/dsda-doom/pull/513/commits/ecc078a8fc2d3425e427dd179811035517fa47bd is not correct, do not change the `matrix.config` section.

Reply:
Oh my bad, I used the "Resolve Conflict" button on github. It changed more than what I thought

Reply:
Now that Im on my pc, umm whats the problem? Can't I add the `macOS x64 Clang` to `matrix.config`?

The commit you linked is just github synching stuff, and it is not really part of the PR

Reply:
> Can't I add the macOS x64 Clang to matrix.config?

You can, but you also changed the syntax to earlier `{ }`. I've used `>-` in MSVC configuration which is not compatible with it.

Reply:
Fix MSVC build:
```diff
diff --git a/.github/workflows/continuous_integration.yml b/.github/workflows/continuous_integration.yml
index e2cea90e6..e28727a7d 100644
--- a/.github/workflows/continuous_integration.yml
+++ b/.github/workflows/continuous_integration.yml
@@ -16,52 +16,49 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {
-              name: "MSVC x64",
-              os: windows-latest,
-              build_type: "Release",
-              extra_options: "-A x64",
-              vcpkg_triplet: x64-windows,
-              shell: bash,
-            }
-          - {
-              name: "MSYS2 UCRT64",
-              os: windows-latest,
-              build_type: "Release",
-              extra_options: "-G Ninja",
-              package_name: "mingw_x64",
-              shell: "msys2 {0}",
-              msystem: ucrt64,
-              msys-env: mingw-w64-ucrt-x86_64,
-              artifact-path: build/*.zip,
-            }
-          - {
-              name: "Linux GCC",
-              os: ubuntu-latest,
-              build_type: "Release",
-              extra_options: "-G Ninja -DCMAKE_INSTALL_PREFIX=/usr",
-              package_name: "linux_gcc",
-              shell: bash,
-              artifact-path: build/*.appimage,
-            }
-          - {
-              name: "macOS arm64 Clang",
-              os: macos-latest,
-              build_type: "Release",
-              extra_options: "-G Ninja",
-              package_name: "mac_arm64",
-              shell: bash,
-              artifact-path: build/*.zip,
-            }
-          - {
-              name: "macOS x64 Clang",
-              os: macos-13,
-              build_type: "Release",
-              extra_options: "-G Ninja",
-              package_name: "mac_x64",
-              shell: bash,
-              artifact-path: build/*.zip,
-            }
+          - name: "MSVC x64"
+            os: windows-latest,
+            build_type: "Release"
+            vcpkg_triplet: x64-windows
+            shell: pwsh,
+            extra_options: >-
+              -A x64
+              -DCMAKE_TOOLCHAIN_FILE="${env:VCPKG_INSTALLATION_ROOT}\scripts\buildsystems\vcpkg.cmake"
+              -DVCPKG_TARGET_TRIPLET=x64-windows
+
+          - name: "MSYS2 UCRT64"
+            os: windows-latest
+            build_type: "Release"
+            extra_options: "-G Ninja"
+            package_name: "mingw_x64"
+            shell: "msys2 {0}"
+            msystem: ucrt64
+            msys-env: mingw-w64-ucrt-x86_64
+            artifact-path: build/*.zip
+
+          - name: "Linux GCC"
+            os: ubuntu-latest
+            build_type: "Release"
+            extra_options: "-G Ninja -DCMAKE_INSTALL_PREFIX=/usr"
+            package_name: "linux_gcc"
+            shell: bash
+            artifact-path: build/*.appimage
+
+          - name: "macOS arm64 Clang"
+            os: macos-latest
+            build_type: "Release"
+            extra_options: "-G Ninja"
+            package_name: "mac_arm64"
+            shell: bash
+            artifact-path: build/*.zip
+
+          - name: "macOS x64 Clang"
+            os: macos-13
+            build_type: "Release"
+            extra_options: "-G Ninja"
+            package_name: "mac_x64"
+            shell: bash
+            artifact-path: build/*.zip
 
     steps:
       - uses: actions/checkout@v4
```

Reply:
I understand now. Done in https://github.com/kraflab/dsda-doom/pull/513/commits/21105ca422f2622cea7e7f3191c3f6cd73f98d4e

Sorry about that 😅